modules/shortcodes/wordads.php

```diff
@@ -21,6 +21,10 @@ function action_init() {
 			return null;
 		}
 		add_shortcode( 'wordads', array( $this, 'wordads_shortcode' ) );
+
+		jetpack_register_block( 'wordads', array(
```

Comment:
We currently use a different function (`jetpack_register_block_type`), but it should be okay to use for now as we should revert to using `jetpack_register_block` soon, as per #11328. We will, however, need to switch to using the prefix for the block name: `jetpack/wordads`

Reply:
Done